helper/impls/helper-impl-110/src/main/java/com/linkedin/avroutil1/compatibility/avro110/Avro110AvscWriter.java

```diff
@@ -85,4 +96,36 @@ private void writeProps(Map<String, Object> props, Jackson2JsonGeneratorWrapper
             delegate.writeObjectField(entry.getKey(), JacksonUtils.toJsonNode(o));
         }
     }
+
+    /**
+     *  Enforces uniform numeric default values across Avro versions
+     */
+    private JsonNode enforceUniformNumericDefaultValues(Schema.Field field) {
+        JsonNode genericDefaultValue = Accessor.defaultValue(field);
+        if (!genericDefaultValue.isNumber()) {
```

Comment:
will this complain and "abort" for fields that are not numeric types?

Reply:
Ah that's right. I've moved the `isNumber` so that it preceeds the call to `enforceUniformNumericDefaultValues`